internal/app/orchestrator/orchestrator.go

```diff
@@ -52,11 +53,15 @@ type Orchestrator interface {
 	// open channels.
 	shutdown(ctx context.Context)
 
+	GetCache() cache.Cache
+
 	GetReadOnlyCache() cache.ReadOnlyCache
 
-	GetDownstreamAggregatedKeys() (map[string]bool, error)
+	GetDownstreamAggregatedKeys() (map[string]bool, marshallable.Error)
```

Comment:
the orchestrator interface should return an error following idiomatic go patterns. The json formatting is only for the purpose of the webapi.